src/v2/controllers/FormController.ts

```diff
@@ -291,13 +291,15 @@ export default Controller.extend({
       if (resp.status === IdxStatus.FAILURE) {
         throw resp.error; // caught and handled in this function
       }
-      // follow idx transaction to render terminal view for session expired error
+
+      // follow idx transaction to render terminal view for terminal responses or session expired error
```

Comment:
nit: revert this file change, since no change happens eventually

Reply:
ah yes good catch thanks